.circleci/config.yml

```diff
@@ -58,3 +64,11 @@ workflows:
           requires:
             - build
             - test
+      - deploy:
+          filters:
+            branches:
+              only: master
+          requires:
+            - build
+            - test
```

Comment:
I think you can only list `compile` here because `compile` already requires `build` and `test`

Reply:
Hmm, that actually seems quite obvious! I wonder why I never tried it before!